src/components/Inbox/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import { Checkbox, Divider, IconButton } from '@material-ui/core';
+import { StylesProvider } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+import './styles.scss';
+
+const Inbox: React.FunctionComponent = () => (
+  <StylesProvider injectFirst>
+    <div className="inbox-container">
+      <header>Inbox</header>
+      <input type="text" placeholder="Search" />
+      <List
+        component="nav"
+        aria-label="unresolved"
+        subheader={(
+          <ListSubheader classes={{ root: 'section-header' }} disableGutters>
+            <IconButton
+              classes={{ root: 'section-header-button' }}
+              aria-label="dropdown"
+              disableRipple
+            >
+              <ArrowDropDownIcon
+                classes={{ root: 'section-header-button-icon' }}
+              />
+            </IconButton>
+            Unresolved
+          </ListSubheader>
+        )}
+      >
+        <Divider classes={{ root: 'divider' }} />
+        <ListItem button classes={{ root: 'list-item' }}>
+          <ListItemIcon classes={{ root: 'icon' }}>
+            <Checkbox
+              classes={{
+                root: 'checkbox',
+                checked: 'checkbox-checked',
+                colorSecondary: 'checkbox-checked',
+              }}
+              size="small"
+              disableTouchRipple
+              color="default"
+            />
+          </ListItemIcon>
+          <ListItemText
+            classes={{
+              root: 'text',
+              primary: 'primary-text',
+              secondary: 'secondary-text',
+            }}
+            primary="Button not working"
+            secondary="The popular times button does nothing"
+          />
+        </ListItem>
```

Comment:
Can we potentially move this to a function to be used with like a `forEach` or something? I have no clue how that'll work but i'd imagine we're gonna need to do this at some point. Also if u wanna keep that for another PR feel free but just wanted to bring it up

Reply:
yea saw some examples online for using a loop in the render to generate the items based on the list that i'll add in a later pr -- was just doing the ui for the cells in this PR